src/signature.c

```diff
@@ -37,7 +37,7 @@
 #include "swupd.h"
 
 #define BUFFER_SIZE   4096
-
+#define CERTNAME "/usr/share/clear/update-ca/ClearLinuxRoot.pem"
```

Comment:
Also, --enable-signature-verification would require this new option (e.g. --with-certificate-path) being set.